e2b/init.sh

```
@@ -3,6 +3,15 @@ set -e
 
 echo "🚀 Initializing E2B container for Claude Code execution..."
 
+# Ensure workspace directory exists and is writable
+if [ ! -w /workspace ]; then
+  echo "⚠️ Warning: /workspace is not writable, attempting to fix permissions..."
+  # Try to fix permissions if running as root or with sudo
+  if [ "$EUID" -eq 0 ] || sudo -n true 2>/dev/null; then
+    sudo chown -R $(id -u):$(id -g) /workspace 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)